### ISSUE TITLE
ReferenceError: _ is not defined on server

### ISSUE THREAD

**Issue by lsunsi** [todda00/meteor-friendly-slugs#18]
I am getting "ReferenceError: _ is not defined" on server when using this package within a local package of mine. I'm probably doing something wrong, but I can't figure out what.

Posts.friendlySlugs('title') is the only related command and removing that line makes the error go away.

Some help?

Edit: I'm using it with SimpleSchema/Collection2.

**Comment by todda00** [comment]
This would be coming from underscore, which friendlySlugs uses and is declared in the Package.onUse:

```
Package.onUse(function(api) {
  api.use(['underscore','coffeescript', 'check','matb33:collection-hooks@0.7.6']);
  api.versionsFrom('1.0');
  api.addFiles('slugs.coffee');
});
```

I am not sure why it wouldn't be pulling in underscore, can you add underscore to your package and see if it resolves this for you?

**Comment by funkybunky** [comment]
running into the same issue. I added underscore to my package, so that shouldn't cause the problem.

**Comment by thekevinbrown** [comment]
Same issue here.

**Comment by thekevinbrown** [comment]
After a bit of digging I think I found what was going on.

I just used `api.use('todda00:friendly-slugs');` in my package.js.

When I looked at .meteor/versions I see:

```
todda00:friendly-slugs@0.1.3
```

Meteor decided it should grab version 0.1.3, not the latest. Typing `meteor update todda00:friendly-slugs` made it pull down 0.3.6, which then worked without this error. I suspect the old version was not ready for meteor 1.0+.
